### DISCUSSION TITLE
Multiple bot accounts for online mode

### DISCUSSION THREAD

**Discussion by jimmy927** [PrismarineJS/mineflayer#3174] [Q&A] [answered]
Assume i want to run a server in online mode.

It seems every account can only login with 1 bot.
Is this configurable in the server maybe ?
Is it possible to add like 20 bots without having 20 full licenses for minecraft ?

**Comment by amoraschi** [comment]
Yes it's configurable, you can turn off online-mode

If I recall correctly, you can join multiple different servers with the same account

**Comment by MrFiend179** [comment, marked as answer]
No, its either only Online Mode of Offline mode, if u wanna run ur server in Online mode then u need premium accounts but if u wanna used cracked accounts u need Offline mode

**Comment by andyl25** [comment]
Theoretically would it be possible with something like a spigot plugin on the server or is it impossible?
